From 01644f14cf1a1eb3ed492abbe9855a0050ada1ee Mon Sep 17 00:00:00 2001
From: "Vasoya,Nikhilx" <nikhilx.vasoya@intel.com>
Date: Mon, 28 Nov 2022 15:58:01 +0530
Subject: [PATCH] Add system_server to readtracefs group

This allows system_server to search/read tracefs entries. It is needed
for attaching cpu timeinstate bpf programs to tracepoints.

Tracked-On: OAM-104011
Signed-off-by: Vasoya,Nikhilx <nikhilx.vasoya@intel.com>
---
 core/java/com/android/internal/os/ZygoteInit.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/os/ZygoteInit.java b/core/java/com/android/internal/os/ZygoteInit.java
index 0f26f57e2155..89b33590ab35 100644
--- a/core/java/com/android/internal/os/ZygoteInit.java
+++ b/core/java/com/android/internal/os/ZygoteInit.java
@@ -783,7 +783,7 @@ public class ZygoteInit {
                 "--setuid=1000",
                 "--setgid=1000",
                 "--setgroups=1001,1002,1003,1004,1005,1006,1007,1008,1009,1010,1018,1021,1023,"
-                        + "1024,1032,1065,3001,3002,3003,3006,3007,3009,3010,3011",
+                        + "1024,1032,1065,3001,3002,3003,3006,3007,3009,3010,3011,3012",
                 "--capabilities=" + capabilities + "," + capabilities,
                 "--nice-name=system_server",
                 "--runtime-args",
-- 
2.38.1

